### ISSUE TITLE
[greater good pledge] request for participation

### ISSUE THREAD

**Issue by vsoch** [incf-nidash/PyNIDM#127]
hey NIDM family!

I'm creating a new initiative, [The Greater Good](https://good-labs.github.io/greater-good-pledge/) that aims to highlight projects with no underlying incentives or ulterior motives - those that are "for the greater good." Many of the nidm projects (I believe) fit strongly within the criteria, and I'm hoping that the community would like to participate. Participation simply means adding the community name to the pledge, and then (optionally) adding a badge to any associated project repos.

Details are provided at the page, and any maintainers / users here are welcome to also join the organization [The Good Labs](https://www.github.com/good-labs) if open source transparency is important to them. It's still early (I'm going to be creating media and other community interactions) and right now I'm getting in touch with projects that I've worked with.

**Comment by dbkeator** [comment]
Hi Vanessa,

I love it!  Thanks for letting us know.

Cheers,
Dave

On Apr 30, 2019, at 2:13 PM, Vanessasaurus <notifications@github.com<mailto:notifications@github.com>> wrote:


hey NIDM family!

I'm creating a new initiative, The Greater Good<https://urldefense.proofpoint.com/v2/url?u=https-3A__good-2Dlabs.github.io_greater-2Dgood-2Dpledge_&d=DwMCaQ&c=dzukdOe-KyRBOwGgecHzPA&r=kLskbFozNRnTkbmV9BT6CbvDF0zO0whkjvyMDS0egZM&m=ZmF0uGGYJhIyeTc0jbgTd4oNM9Hs3jYYTLrem1oyh00&s=6jTFaS6zWvC6nOazcWC2tenH84n8cwCiE-Ahs-F3RBw&e=> that aims to highlight projects with no underlying incentives or ulterior motives - those that are "for the greater good." Many of the nidm projects (I believe) fit strongly within the criteria, and I'm hoping that the community would like to participate. Participation simply means adding the community name to the pledge, and then (optionally) adding a badge to any associated project repos.

Details are provided at the page, and any maintainers / users here are welcome to also join the organization The Good Labs<https://urldefense.proofpoint.com/v2/url?u=https-3A__www.github.com_good-2Dlabs&d=DwMCaQ&c=dzukdOe-KyRBOwGgecHzPA&r=kLskbFozNRnTkbmV9BT6CbvDF0zO0whkjvyMDS0egZM&m=ZmF0uGGYJhIyeTc0jbgTd4oNM9Hs3jYYTLrem1oyh00&s=9jiFb137_gYGDO9gDzltjghiYQRnpI2rfRyMd19ZRoM&e=> if open source transparency is important to them. It's still early (I'm going to be creating media and other community interactions) and right now I'm getting in touch with projects that I've worked with.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_incf-2Dnidash_PyNIDM_issues_127&d=DwMCaQ&c=dzukdOe-KyRBOwGgecHzPA&r=kLskbFozNRnTkbmV9BT6CbvDF0zO0whkjvyMDS0egZM&m=ZmF0uGGYJhIyeTc0jbgTd4oNM9Hs3jYYTLrem1oyh00&s=SLWaqJiJSZaBguJLPGR8j_MwYkp2uk7XNB1Ji210s7g&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AAKC3Y4S3AUC6OQ2RKLGZ2DPTCYYBANCNFSM4HJQGUCA&d=DwMCaQ&c=dzukdOe-KyRBOwGgecHzPA&r=kLskbFozNRnTkbmV9BT6CbvDF0zO0whkjvyMDS0egZM&m=ZmF0uGGYJhIyeTc0jbgTd4oNM9Hs3jYYTLrem1oyh00&s=DSpsJ69IZWZ38b-GLd84tX2huAS18HZn104vCBHw_gU&e=>.

________________________________

This message may contain confidential information and is for the sole use of the intended recipient(s). If you are not the intended recipient, do not use, distribute, or copy this e-mail. Please notify the UC Irvine Health – Compliance and Privacy Office via email at hacompliance@uci.edu or by phone 888-456-7006 immediately if you have received this e-mail in error. E-mail transmission cannot be guaranteed to be secure or error-free as information could be intercepted, corrupted, lost, destroyed, arrive late or incomplete, or contain viruses. The sender therefore does not accept liability for any errors or omissions in the contents of this message, which arise as a result of e-mail transmission.

**Comment by vsoch** [comment]
of course! I'm going to get some FAQ and participation scenarios up today - for example, I realize that it might be easy to make the assumption that any projects bringing in money don't quality, but it's not about that. I'll share these when they are ready!

**Comment by vsoch** [comment]
FAQ is ready to go! https://good-labs.github.io/greater-good-pledge/faq/

Let me know if you have other questions, or want to give feedback on the criteria. I haven't created versions/releases yet because I expect some work might be needed on the criteria itself. You can ask here, or post an issue at https://github.com/good-labs/greater-good-pledge/issues. I want to provide support and signal for these kinds of projects that are for the greater good, so this initiative is important.
